gossip/consensus_callbacks.go

```diff
@@ -129,6 +130,21 @@ func (s *Service) applyNewState(
 
 	// Process EVM txs
 	block, evmBlock, totalFee, receipts := s.executeEvmTransactions(block, evmBlock, statedb)
+	for _, receipt := range receipts {
+		for _, log := range receipt.Logs {
+			if len(log.Topics) == 0 {
+				continue
+			}
+
+			// temprorary magic. will be placed to a constant
```

Comment:
line 139: spelling
line 145: remove empty line
line 143: sealEpoch = true. Should break the loops (line 133,134)?